conn-ntg/README.md

```diff
@@ -0,0 +1,58 @@
+# Connect Native Trading Gateway (NTG)
+
+## Available service types
+
+This "Connect" box provides users with the following services:
+
+### NTG Client
+
+The configuration type that needs to be used is - **th2_service:NTGClient**.
+
+#### Configuration parameters
+
+```yaml
+# Real time IP address
+serverIP: "127.0.0.1"
+
+# Real time connection port
+serverPort: 8181
+
+# Connect timeout in milliseconds.
+connectTimeout: 30000
+
+# Login timeout in milliseconds.
+loginTimeout: 30000
+
+# Logout timeout in milliseconds.
+logoutTimeout: 5000
+
+# Idle timeout in milliseconds (if no message has been sent then client must sent Heartbeat message)
```

Comment:
# Idle timeout in milliseconds (if no message has been sent then **the** client must sent **a** Heartbeat message)

Reply:
done